specification/introduction/introduction.tex

```diff
@@ -8,73 +8,69 @@ \section{Design principles}
+    \item[Nodes can exchange long payloads] --- Nodes must be provided with a simple way of exchanging large
     data structures that cannot fit into a single transport frame (such as GNSS solutions, 3D vectors, etc.).
     UAVCAN should perform automatic transfer decomposition and reassembly at the protocol level,
     hiding the related complexity from the application.
 
-    \item[Support for redundant interfaces and redundant nodes] - UAVCAN must be suitable for use in
+    \item[Support for redundant interfaces and redundant nodes] --- UAVCAN must be suitable for use in
     applications that require modular redundancy.
 
-    \item[High throughput, low latency communication] - UAVCAN must be suitable for use in
+    \item[High throughput, low latency communication] --- UAVCAN must be suitable for use in
     hard real-time systems.
 
-    \item[Simple logic, low computational requirements] - UAVCAN targets a wide variety of embedded systems,
+    \item[Simple logic, low computational requirements] --- UAVCAN targets a wide variety of embedded systems,
     from high-performance embedded on-board computers for intensive data processing
     (e.g., a high-performance GNU/Linux-powered machine) to extremely resource-constrained microcontrollers.
     The latter imposes severe restrictions on the amount of logic needed to implement the protocol.
 
-    \item[Common high-level functions must be clearly defined] - UAVCAN defines standard services
+    \item[Common high-level functions must be clearly defined] --- UAVCAN defines standard services
     and messages for common high-level functions, such as network discovery, node configuration,
     node software update, node status monitoring (which naturally grows into a vehicle-wide health monitoring),
     network-wide time synchronization, dynamic node ID allocation (a.k.a. plug-and-play node support), etc.
 
-    \item[Open specification and reference implementations] - The UAVCAN specification is open and
+    \item[Open specification and reference implementations] --- The UAVCAN specification is open and
     freely available; the reference implementations are distributed under the terms of the permissive MIT License.
 \end{description}
 
 \section{Capabilities}
 
-This section summarizes the capabilities of the UAVCAN protocol.
-
 UAVCAN-based networks can accommodate up to 127 nodes on the same logical bus.
-More nodes can be added to the system by separating the bus into several independent logical segments
-interconnected via gateway nodes.
 
-UAVCAN supports up to 32768 distinct message data types and up to 256 distinct service data types.
-Part of those are reserved for the standard data types defined by the specification;
-the rest are available for vendor- and application-specific data types.
-More information is provided in the chapter \ref{sec:application_layer}.
+UAVCAN supports up to 65536 message, 256 service, and 8 anonymous data types.
```

Comment:
I suggest a different wording:

> and at least 8 anonymous data types.

"at least" works better because the transport layer spec says that the subset of message type ID usable with anonymous transfers is dependent on the transport layer in use.

Reply:
👌